src/mongoose_scram.erl

```diff
@@ -230,3 +242,16 @@ do_check_digest([{Sha,_Prefix} | RemainingSha], ScramMap, Digest, DigestGen, Pas
 supported_sha_types() ->
     [{sha,      <<?SCRAM_SHA_PREFIX>>},
      {sha256,   <<?SCRAM_SHA256_PREFIX>>}].
+
+configured_sha_types(Host) ->
+    PasswordFormat = ejabberd_auth:get_opt(Host, password_format),
+    ScramSha = proplists:get_value(scram, [PasswordFormat]),
+    case ejabberd_auth:get_opt(Host, password_format) of
+        Scram when Scram == scram orelse Scram == {scram} ->
```

Comment:
When is it possible to have `{scram}` as a value of the config option?

Reply:
In general it should not be possible but I figured that it could be common typo when someone has {scram, [sha256]} and decides to go back to just scram leaving {scram}. It doesn't cost us much but the config would be more forgiving with the syntax typo.